src/com/yetanalytics/lrs_admin_ui/views/menu.cljs

```diff
@@ -16,8 +16,7 @@
     [menu-item {:name @(subscribe [:lang/get :header.nav.credentials]) :page :credentials}]
     (when @(subscribe [:oidc/show-account-nav?])
       [menu-item {:name @(subscribe [:lang/get :header.nav.accounts]) :page :accounts}])
-    (when @(subscribe [:db/get-stmt-html-enabled])
-      [menu-item {:name @(subscribe [:lang/get :header.nav.browser]) :page :browser}])
+    [menu-item {:name @(subscribe [:lang/get :header.nav.browser]) :page :browser}]
```

Comment:
That means we can remove the `:db/get-stmt-html-enabled` sub and the `::db/enable-statement-html` field in app-db, as well as the `LRSQL_ENABLE_STMT_HTML` env var in the backend, right?

Reply:
No we need to keep the ENV VAR. This whole thing is because a client has an old xapi client lib that accidentally asks for HTML, so we need to TURN IT OFF but still show statement browser. The entire scope of this change is just that the menu ignores the setting, not that the setting goes away.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/78/commits/b4b1bb1c08dc966d7455da5dd3ee64cff576e9fa

But you are right the frontend doesnt need to know